docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -128,7 +127,7 @@ The output should look like:
 
 ```
 [kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.
-[init] Using Kubernetes version: v1.7.0
+[init] Using Kubernetes version: v1.8.0
 [init] Using Authorization modes: [Node RBAC]
```

Comment:
I'm pretty sure the command output for both init/join changed quite a lot. Please do an e2e run and paste logs here

Reply:
Thanks, I forgot how much had changed but you're right.